model/gen-ai/registry.yaml

```
@@ -474,3 +474,23 @@ groups:
                   "finish_reason": "stop"
                 }
               ]
+      - id: gen_ai.evaluation.name
+        stability: development
+        type: string
+        brief: The name of the evaluation used for the GenAI response.
```

Can we be more specific?

```suggestion
        brief: The name of the evaluation metric used for the GenAI response.
```

metric | method | etc